src/node.cc

```diff
@@ -3585,6 +3593,8 @@ static void PrintHelp() {
          "  --v8-pool-size=num    set v8's thread pool size\n"
 #if HAVE_OPENSSL
          "  --tls-cipher-list=val use an alternative default TLS cipher list\n"
+         "  --ssl-[no-]def-store  use only default certificate store\n"
```

Comment:
The default certificate store *is* the bundled one. Something like this would make more sense:

```
--use-ssl-ca-store    use the system's OpenSSL CA certificate store instead of the compiled in Mozilla roots
```